glass/__init__.py

```diff
@@ -1,4 +1,68 @@
 try:
-    from ._version import __version__, __version_tuple__  # noqa: F401
+    from glass._version import __version__, __version_tuple__
```

Comment:
Isn't version one place where you don't do an absolute import? At least in my experience, but can't find a resource to back this up

Reply:
I use absolute for my projects and it works fine, but I can update this if relative is recommended for versions imports.

Reply:
Looked around a bit - [`setuptools_scm` suggests using `importlib.metadata`](https://setuptools-scm.readthedocs.io/en/latest/usage/#python-package-metadata) to obtain version in `__init__` file, but scientific-python/cookie suggests doing [`from ._version import version as __version__`](https://github.com/scientific-python/cookie/blob/99acb847958802c719e05cca26ffcb3ddc9e47e1/%7B%7Bcookiecutter.project_name%7D%7D/src/%7B%7Bcookiecutter.__project_slug%7D%7D/__init__.py#L11) (probably because this works with majority of the build-backends).

Reply:
I'd probably lean toward the `scientif-python` one, I must have found that convention somewhere!